src/utils/sammonStress.ts

```diff
@@ -0,0 +1,33 @@
+export const getSammonStress = (rawData: any[], reductedData: any[]) => {
```

Comment:
Try to not use `any` in those helper functions. `any` is ok if you provide some unit tests :wink: